client/package.json

```diff
@@ -28,6 +28,7 @@
     "axios": "0.27.2",
     "cookie": "0.5.0",
     "csvtojson": "2.0.10",
+    "eslint-plugin-jest": "^27.0.4",
```

Comment:
could you please move it to devDeps?

We install dependencies inside container and try to keep them small.

Reply:
Sure. Fixed